### PR TITLE
chore(dashboards): Fix formatting issue

### PR DESCRIPTION
**Pull request by austin-schaefer**
Fixes some formatting issues introduced in #16900

### CODE REVIEW

**Comment by austin-schaefer** [comment]
@homelessbirds I'd love if you can review this one since you were a reviewer on original PR and have context.

File: src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx [hunk 1]
```diff
@@ -405,24 +405,27 @@ Exporting to PDF or as an image has certain limitations depending on the chart t
...
-		
 
 In order to minimize disruption, customers who want to enable this functionality can create custom roles and assign them to their users prior to April 30, 2024. After April 30, 2024, this function will be generally available, and these capabilities will activate.
 
-**Note:** The ability to create “Live chart URLs” will be removed from <DoNotTranslate>HIPAA</DoNotTranslate> and <DoNotTranslate>Fedramp</DoNotTranslate> accounts.
+<Callout variant="important">
+The ability to create live chart URLs will be removed from <DoNotTranslate>HIPAA</DoNotTranslate> and <DoNotTranslate>Fedramp</DoNotTranslate> accounts.
```
Comment (bradleycamacho) [inline src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx:412]: Probably out of scope, but this confused me. I thought there was a piece missing of "will be removed on EOL_DAY_HERE" - it took me a second to realize it meant the feature is locked for HIPAA and Fedramp accounts

Comment (bradleycamacho) [inline src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx:412]:
```suggestion
The ability to create live chart URLs will be removed from <DoNotTranslate>HIPAA</DoNotTranslate> and <DoNotTranslate>FedRAMP</DoNotTranslate> accounts.
```

Comment (bradleycamacho) [reply src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx:412]: I believe this is right - https://www.fedramp.gov/

File: src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx [hunk 2]
```diff
@@ -405,24 +405,27 @@ Exporting to PDF or as an image has certain limitations depending on the chart t
...
+</Callout>
 
-### <DoNotTranslate>Live Chart Url (self)</DoNotTranslate> 
+### <DoNotTranslate>Live Chart URL (self)</DoNotTranslate>
```
Comment (bradleycamacho) [inline src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx:415]: Should these be title case and DNT'ed? Gonna suggest they be removed, but feel free to merge w/out implementing those

Comment (bradleycamacho) [inline src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx:415]:
```suggestion
### Live chart URL (self) [#live-chart-self]
```

File: src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx [hunk 3]
```diff
@@ -405,24 +405,27 @@ Exporting to PDF or as an image has certain limitations depending on the chart t
...
-View: The user can see the live chart URLs they have created. 
-Modify: The user can create a live chart URL. 
-Delete: The user can delete any live chart URL they have created.
 
-### <DoNotTranslate>Live Chart Url (all)</DoNotTranslate>
+* View: The user can see the live chart URLs they have created. 
+* Modify: The user can create a live chart URL. 
+* Delete: The user can delete any live chart URL they have created.
+
+### <DoNotTranslate>Live Chart URL (all)</DoNotTranslate>
```
Comment (bradleycamacho) [inline src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx:422]:
```suggestion
### Live chart URL (all) [#live-chart-all]
```

File: src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx (hunk 2 above)
Comment (bradleycamacho) [inline src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx:415]: Should these be documented here or in the user management docs?

**Review by bradleycamacho — CHANGES_REQUESTED** [review]

File: src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx (hunk 1 above)
Comment (austin-schaefer) [reply src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx:412]: Agree, this confused me too. I'll ask Jeff to clarify this when he's back.

File: src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx (hunk 2 above)
Comment (austin-schaefer) [reply src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx]: Probably user management

**Review by bradleycamacho — DISMISSED** [review]

**Review by jeff-colucci — APPROVED** [review]

**Comment by jeff-colucci** [comment]
Built successfully at https://app.netlify.com/sites/docs-website-netlify/deploys/66297fe9a409980558a1924a

**Pull request merged**